From c4514a64db5a237fb2b32e51d352b6b22b8bfc4f Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Thu, 13 Oct 2022 15:07:22 +0800
Subject: [PATCH 43/44] colors_device_defaults: Add chroma to light surface
 highlight color

With dynamic Monet themes, most of the UI has at least a small amount
of color (chroma), so pure white stands out as the sole outlier (e.g.
Settings search bar in light mode). Use the 99% lightness shade instead
to improve consistency.

Change-Id: I7dfe3e483fa207ce4d77349ac97872e0831ef3c8
Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 core/res/res/values/colors_device_defaults.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 4c5a008a3402..5d3d6a3155a9 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -59,7 +59,7 @@
     <color name="surface_variant_dark">@color/system_neutral1_700</color>
     <color name="surface_variant_light">@color/system_neutral2_100</color>
     <color name="surface_dark">@color/system_neutral1_800</color>
-    <color name="surface_highlight_light">@color/system_neutral1_0</color>
+    <color name="surface_highlight_light">@color/system_neutral1_10</color>
 
     <!-- Please refer to text_color_[primary]_device_default_[light].xml for text colors-->
     <color name="foreground_device_default_light">@color/text_color_primary_device_default_light</color>
-- 
2.34.1

